pkg/sabactl/cmd/machines.go

```diff
@@ -40,9 +45,22 @@ var machinesGetCmd = &cobra.Command{
 			if err != nil {
 				return err
 			}
-			e := json.NewEncoder(cmd.OutOrStdout())
-			e.SetIndent("", "  ")
-			return e.Encode(ms)
+			if ok && *outputFormat == "simple" {
```

Comment:
I prefer to separate `output` option into a separate PR.
However, in this case it is not a problem.